frontend/app_flowy/lib/workspace/presentation/widgets/menu/widget/app/section/item.dart

```diff
@@ -44,7 +44,7 @@ class ViewSectionItem extends StatelessWidget {
             onTap: () => onSelected(context.read<ViewBloc>().state.view),
             child: FlowyHover(
               config: HoverDisplayConfig(hoverColor: theme.bg3),
-              builder: (_, onHover) => _render(context, onHover, state),
+              builder: (_, onHover) => _render(context, onHover, state, theme.textColor),
```

Comment:
I think textColor is only for text. Here would be icon color?

Reply:
Fixed and created a new value "iconColor"